packages/aikit-ui/src/blocks/Chat.tsx

```diff
@@ -0,0 +1,300 @@
+        <Button
+          variant="ghost"
+          size="icon"
+          onClick={handleReset}
+          aria-label="Clear conversation"
+          title="Clear conversation"
+        >
+          <IterationCcw size={20} color="#525252" />
+        </Button>
+      </div>
+
+      {/* Live region for status updates */}
+      <div aria-live="polite" aria-atomic="true" className="sr-only">
+        {isLoading && "AI is thinking..."}
+      </div>
+
+      {/* Conversation area */}
+      <Conversation>
+        <ConversationContent
+          role="log"
+          aria-label="Chat conversation"
+          tabIndex={-1}
+        >
+          {messages.map((message) => (
+            <Message
+              from={message.role}
+              key={message.id}
+              tabIndex={-1}
+              role="article"
+              aria-label={`Message from ${message.role}`}
+            >
+              <MessageContent>
+                <div>{renderMessagePart(message.parts[0])}</div>
+              </MessageContent>
+            </Message>
+          ))}
+          {isLoading && (
+            <Message from="assistant" key="loading">
+              <MessageContent>
+                <div aria-live="polite">
+                  <span className="animate-pulse">Thinking...</span>
+                </div>
+              </MessageContent>
+            </Message>
+          )}
+        </ConversationContent>
+        <ConversationScrollButton />
+      </Conversation>
+
+      {/* Input area */}
+      <PromptInput
+        onSubmit={handleSubmit}
+        className="relative mt-4"
+        role="form"
+        aria-label="Chat input form"
+      >
+        <PromptInputTextarea
+          ref={textareaRef}
+          id="chat-input"
+          onChange={handleInputChange}
+          onKeyDown={(e) => {
+            if (e.key === "Escape") {
+              if (input) {
+                setInput("");
+              } else {
+                textareaRef.current?.blur();
+              }
+            }
+          }}
+          value={input}
+          disabled={isLoading}
+          placeholder="Start prompting..."
+          aria-label="Chat message input"
+          aria-describedby="input-help"
+          rows={1}
+        />
+
+        {/* Hidden help text */}
+        <div id="input-help" className="sr-only">
+          Press Enter to send message, Shift+Enter for new line, Escape to clear
+        </div>
+
+        <PromptInputToolbar>
+          <PromptInputTools>
+            {/* Model selector - show hint when no models available */}
+            {modelsConfig.models.length === 0 ? (
+              <div className="text-muted-foreground px-3 py-2 text-sm">
+                No models available
+              </div>
+            ) : (
+              <PromptInputModelSelect
+                onValueChange={handleModelChange}
+                value={currentModelAlias}
+                disabled={isLoading || modelsConfig.models.length === 0}
```

Comment:
This code never executes when modelsConfig.models.length === 0. Check can be removed.